src/main/java/io/cdap/plugin/dlp/Utils.java

```diff
@@ -172,18 +174,34 @@ public static StructuredRecord getStructuredRecordFromTable(Table table, Structu
   }
 
   /**
-   * Creates a DLP Table from a StructuredRecord
+   * Creates a DLP Table using the all fields from the given StructuredRecord
    *
    * @param record StructuredRecord to convert to Table
    * @return Table with one row containing the data from {@code record}
    * @throws Exception when record contains unexpected type or type-casting fails due to invalid values
    */
   public static Table getTableFromStructuredRecord(StructuredRecord record) throws Exception {
+    return getTableFromStructuredRecord(record, null);
+  }
+
+  /**
+   * Creates a DLP Table using the required fields from StructuredRecord
+   *
+   * @param record         StructuredRecord to convert to Table
+   * @param requiredFields Set of fields to include in the conversion, pass empty set to include all fields
+   * @return Table with one row containing the data from {@code record}
+   * @throws Exception when record contains unexpected type or type-casting fails due to invalid values
+   */
+  public static Table getTableFromStructuredRecord(StructuredRecord record, Set<String> requiredFields)
```

Comment:
Is the second parameter nullable? If it is, annotate it with `@Nullable`.

Reply:
done